### PR TITLE
Splitting Horizontal and Vertical Layouts

### LINKED ISSUE #1616: Add separate pages for Vertical and Horizontal layouts
Using the left-side menu to find the instructions for Vertical and Horizontal layouts takes longer than it needs to because those components are included under the [Basic Layouts page](https://vaadin.com/docs/latest/components/basic-layouts). For better discoverability, split this page into two separate ones for Vertical and Horizontal layouts.

### LINKED ISSUE #2106: Simplify layout examples
**Vote to prioritize:** add a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to help the community and maintainers prioritize this issue.

The dynamic/interactive examples in the Basic Layouts documentation (i.e. for Horizontal and Vertical Layout) make it difficult to learn how to configure layouts in your own code.

While the interactive examples can be useful, it would be beneficial to also give as simple as possible, static code examples of alignment, justification, spacing, etc.

### PR DESCRIPTION
**Pull request by Mikaelsu**
Moved Horizontal and Vertical Layouts to separate chapters and wrote new examples. 

Closes #2106
Closes #1616

### CODE REVIEW

**Review by rolfsmeds — CHANGES_REQUESTED** [review]
This PR should also
- Remove the Basic Layouts page and replace the links to it from:
- articles/components/index.asciidoc
- artitlces/components/scroller/index.asciidoc
- articles/styling/styling-other-elements.adoc
- articles/styling/styling-components/index.adoc
- articles/styling/styling-components/styling-component-instances.adoc
- articles/tools/designer/using-designer/layout.asciidoc

File: frontend/demo/component/basiclayouts/basic-layouts-horizontal-layout-individual-alignment.ts [hunk 1]
```diff
@@ -21,55 +19,23 @@ export class Example extends LitElement {
...
-  private alignFirstItem = 'auto';
-
   protected override render() {
     return html`
+    <!-- tag::snippet[] -->
       <vaadin-horizontal-layout
         theme="spacing padding"
-        class="height-5xl"
-        style="align-items: ${this.alignLayoutItems}"
+        class="height-5xl" // hidden-source-line
```
[the remark this answers is not in the document]
Comment (Mikaelsu) [reply frontend/demo/component/basiclayouts/basic-layouts-horizontal-layout-individual-alignment.ts]: Agreed! It looks better with `4xl`. Pushing an update...

File: frontend/demo/component/basiclayouts/basic-layouts-horizontal-layout-individual-alignment.ts [hunk 2]
```diff
@@ -21,55 +19,23 @@ export class Example extends LitElement {
...
+        style="align-items: center"
       >
         <vaadin-text-area
           label="Text area 1"
-          style="align-self: ${this.alignFirstItem}"
-        ></vaadin-text-area>
+          style="align-self: end"
+        >
+        </vaadin-text-area>
         <vaadin-text-area label="Text area 2"></vaadin-text-area>
```
[the remark this answers is not in the document]
Comment (Mikaelsu) [reply frontend/demo/component/basiclayouts/basic-layouts-horizontal-layout-individual-alignment.ts:35]: Thanks, @rolfsmeds ! I pushed an update, please review

[the remark this answers is not in the document]
Comment (rolfsmeds) [reply frontend/demo/component/basiclayouts/basic-layouts-horizontal-layout-individual-alignment.ts:35]: Looks good!

**Review by rolfsmeds — APPROVED** [review]

**Pull request merged**